src/app/common/directives/oc-reorder/js/oc-reorder.modal.service.js

```diff
@@ -0,0 +1,121 @@
+angular.module('orderCloud')
+    .factory('ocReorderModal', ocReorderModalService)
+;
+
+function ocReorderModalService($q, toastr, $exceptionHandler, OrderCloudSDK, ocLineItems, $uibModal) {
```

Comment:
Yeah, I was thinking the same thing but wasn't sure what else to call it. ocReorder is the directive name now. ocReorderService seems weird, especially on unit tests